cmd/verify_metadata.go

```diff
@@ -87,28 +86,35 @@ func verifyMetadata(cmd *cobra.Command, args []string) error {
         return err
     }
 
-    disableDeletes, err := cmd.Flags().GetBool(disableDeletesFlagName)
+    artifactPath, err := cmd.Flags().GetString(artifactPathFlagName)
     if err != nil {
         return err
     }
 
-    metadata_type, err := cmd.Flags().GetString(metadataTypeFlagName)
+    disableDeletes, err := cmd.Flags().GetBool(disableDeletesFlagName)
     if err != nil {
         return err
     }
-    if metadata_type != "buildinfo" && metadata_type != "vexinfo" && metadata_type != "healthinfo" {
-        return fmt.Errorf("Metadata type should be either of buildinfo, vexinfo, healthinfo")
-    }
 
     serviceAccountKeyFilePath, err := cmd.Flags().GetString(serviceAccountKeyFilePathFlagName)
     if err != nil {
         return err
     }
+
+    metadata_type, err := cmd.Flags().GetString(metadataTypeFlagName)
+    if err != nil {
+        return err
+    }
+
+    if metadata_type != "buildinfo" && metadata_type != "vexinfo" && metadata_type != "healthinfo" && metadata_type != "premiuminfo" {
```

Comment:
we can declare an array, and then check if metadata_type is present in that array or not.

Reply:
done